acct-user/vdr/vdr-0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="VDR (VideoDiskRecorder) service user"
+ACCT_USER_ID=452
+ACCT_USER_HOME=/var/lib/vdr
+ACCT_USER_HOME_OWNER=vdr:vdr
+# note: the first group name in the array will become the primary group
+ACCT_USER_GROUPS=( vdr audio cdrom video )
+IUSE="graphlcd legacy-homedir remote serial systemd"
+
+acct-user_add_deps
+
+pkg_setup() {
+	# if user wants to preserve his existing vdr installation, he can set
+	# USE=legacy-homedir
+	use legacy-homedir && ACCT_USER_HOME=/var/vdr
+
+	# media-plugins/vdr-graphlcd
+	use graphlcd && ACCT_USER_GROUPS+=( lp usb )
+
+	# media-plugins/vdr-remote, _only_ when systemd is installed
+	if use remote; then
+		use systemd && ACCT_USER_GROUPS+=( input ) || \
+			ewarn "\nuse-flag remote only applicable on systemd systems\n"
+	fi
+
+	# media-plugins/vdr-serial: add group to access /dev/ttyS*
+	# on systemd systems: add "dialout", non-systemd systems: add "uucp"
+	if use serial; then
+		use systemd && ACCT_USER_GROUPS+=( dialout ) || ACCT_USER_GROUPS+=( uucp )
+	fi
+}
+
+pkg_preinst() {
+	# if useflag legacy-homedir is _not_ set, check if user vdr exists and what his homedir is
+	if ! use legacy-homedir; then
+		local EXISTING_HOME=$(egethome vdr)
```

Comment:
Does this work? IIRC, you need to inherit ```user-info``` to get the command working?

Reply:
Yes that works, I have tested it. `acct-user.eclass` inherits `user.eclass` which inherits `user-info.eclass`.

Reply:
Please don't rely on indirect inherits.

Reply:
ok, I add user-info.eclass